src/tools/simple-kernel-timer/kp_kernel_info.h

```diff
@@ -5,9 +5,28 @@
 #include <stdio.h>
 #include <sys/time.h>
 #include <cstring>
+
+#if defined(__GXX_ABI_VERSION)
```

Comment:
The function works fine with Clang as well.  I haven't tested with other compilers.